src/main/java/friday/util/Parser.java

```diff
@@ -0,0 +1,196 @@
+package friday.util;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import friday.command.AddCommand;
+import friday.command.Command;
+import friday.command.DeleteCommand;
+import friday.command.ExitCommand;
+import friday.command.FindCommand;
+import friday.command.InvalidCommand;
+import friday.command.ListCommand;
+import friday.command.MarkCommand;
+import friday.command.OnCommand;
+import friday.command.UnmarkCommand;
+import friday.task.Deadline;
+import friday.task.Event;
+import friday.task.Todo;
```

Comment:
```suggestion
import friday.task.*
```

Would it be better to import all since you are using all of them anyways?

Reply:
No, it contradicts the style guide so you shouldn't do that?